docs/setup/mesos.md

```diff
@@ -222,6 +222,8 @@ When running Flink with Marathon, the whole Flink cluster including the job mana
 
 `mesos.initial-tasks`: The initial workers to bring up when the master starts (**DEFAULT**: The number of workers specified at cluster startup).
 
+`mesos.constraints.hard.hostattribute`: Constraints for task placement on mesos (**DEFAULT**: None).
```

Comment:
The description seems too generic; I suggest `Constraints for task placement on mesos based on host attributes`.   An inline example would also be nice, or at least some description of the value format.